src/common/time/ESSTime.cpp

```diff
@@ -11,6 +11,7 @@
 // #define TRC_LEVEL TRC_L_DEB
 
 #include <common/time/ESSTime.h>
+#include <cstdint>
 #include <inttypes.h>
```

Comment:
If you use <cinttypes> you will automatically get <cstdint> so this could be a single include